ndarray/src/main/java/org/tensorflow/ndarray/impl/sparse/AbstractSparseNdArray.java

```diff
@@ -402,6 +402,26 @@ public boolean equals(Object obj) {
     return values.equals(other.values);
   }
 
+  /**
+   * A String showing the type, default value, number of elements and
+   * the dense shape of this sparse ndarray.
+   * @return A string containing the type, default value, number of elements and shape.
+   */
+  @Override
+  public String toString() {
+    long numElements = values == null ? 0 : values.size();
+    String strDefault;
+    if (defaultValue == null) {
+      strDefault = "<null>";
+    } else if (defaultValue instanceof Number){
+      strDefault = defaultValue.toString();
+    } else {
+      strDefault = "'" + defaultValue + "'";
+    }
+    return this.getClass().getSimpleName() + "(defaultValue=" + strDefault
```

Comment:
I think the `this.getClass().getSimpleName()` will return strange names when it comes to tensors, like `TensorMapper.DenseTFloat32Impl` or something like that.

Maybe we should always return the class name of the `type` instead?

Reply:
Yeah, I plan to check to see what the output is in TF and then override it if it's nasty.